Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -9917,6 +9918,1640 @@
+													"    \"schoolTypeDescriptor\",\r",
+													"    \"administrativeFundingControlDescriptor\",\r",
+													"    \"addresses\",\r",
+													"    \"schoolCategories\"\r",
+													"];\r",
+													"\r",
+													"pm.test(\"The response model should contain the id and the primary key properties\",\r",
+													"    () => {\r",
+													"        _.each(keys, (key) => {\r",
+													"            pm.expect(originalEntity[key], key).to.deep.equals(profileEntity[key]);\r",
+													"        })\r",
+													"});\r",
+													"\r",
+													"pm.test(\"The response model should contain the explicitly included properties\",\r",
+													"    () => {        \r",
+													"        // Traverse all properties that are not pripary keys\r",
+													"        for (const [key, value] of Object.entries(originalEntity).filter(function (e) { return this.indexOf(e[0]) < 0; }, keys)) \r",
+													"        {     \r",
+													"            // If property is not an array, make a deep comparison\r",
+													"            if (!Array.isArray(originalEntity[key])) {\r",
+													"                if(profileIncludeKeys.indexOf(key) > -1){\r",
+													"                    pm.expect(profileEntity[key], key).to.deep.equal(originalEntity[key]);\r",
+													"                }\r",
+													"                else {\r",
+													"                    pm.expect(profileEntity[key], key).to.be.undefined;\r",
+													"                }\r",
+													"            }\r",
+													"            // if property is an array, exclude empty arrays\r",
+													"            else if (originalEntity[key].length !== 0){\r",
+													"                if(profileIncludeKeys.indexOf(key) > -1){\r",
+													"                    pm.expect(profileEntity[key], key).to.deep.equal(originalEntity[key]);\r",
+													"                }\r",
+													"            }\r",
+													"        }  \r",
+													"});\r",
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "GET",
+										"header": [
+											{
+												"key": "Accept",
+												"value": "application/vnd.ed-fi.school.test-profile-resource-includeonly.readable+json",
+												"type": "text"
+											}
+										],
+										"url": {
+											"raw": "{{ApiBaseUrl}}/data/v3/ed-fi/schools/{{known:schoolId}}",
+											"host": [
+												"{{ApiBaseUrl}}"
+											],
+											"path": [
+												"data",
+												"v3",
+												"ed-fi",
+												"schools",
+												"{{known:schoolId}}"
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "The Read content only includes certain properties from derived association",
```

Comment:
Add missing assertion "the number of properties on the response model should reflect the number of included properties plus the primary key properties"